package.json

```diff
@@ -47,8 +47,9 @@
     ],
     "all": true
   },
-
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "7.10.4",
```

Comment:
this should be dev dependency

Reply:
Okay. I will add this as dev dependency.